elements/designer-mini-nav/designer-mini-nav.html

```diff
@@ -47,6 +47,46 @@
 
   Polymer({
     is: 'designer-mini-nav',
+
+    properties: {
+
+      expanded: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true,
+      },
+
+      _collapsing: {
+        type: Boolean,
+        value: function() { return false; },
+      },
+
+    },
+
+    listeners: {
+      'designer-mini-nav-collapse': '_onCollapse',
```

Comment:
It seems like a property of a good set of drag-drop behaviors to provide a generic event to use for this purpose.